Create rdm-article.md (WIP) (#2265)
alan-turing-institute/the-turing-way: book/website/reproducible-research/rdm/rdm-article.md

(rr-rdm-article)=
# Sharing your data through a Data Article

> A Data Article (also known as a Data Paper/Note/Release, or Database article) is a publication that is focused on the description of a dataset. It makes use of the traditional journal article structure, but focuses on the data collection and methodological aspects and generally not on the interpretation or discussion of the results.

Reviewer: ```suggestion
A Data Article (also known as a Data Paper/Note/Release, or Database article) is a publication that is focused on the description of a dataset.
It uses the traditional journal article structure, but focuses on the data-collection and methodological aspects and generally not on the interpretation or discussion of the results.
```

(rr-rdm-article-motivations)=
##  Motivations For Sharing Data through a Data Article

> Data articles are in line with the {ref}`FAIR principles<rr-rdm-fair>`, especially since most publishers will encourage you to share the data through a data repository. The benefit of a Data Article is that your output will be {ref}`peer reviewed<cm-peer-review>`, something which is generally not the case for datasets that are archived on {ref}`data repositories<rr-rdm-sharing>`. It also facilitates recognition for datasets through research assessment procedures that are more traditionally focused on publication output. Publishing a data paper will therefore increase the visibility, credibility and usability of the data.

Reviewer 2: ```suggestion
Data articles are in line with the {ref}`FAIR principles<rr-rdm-fair>`, especially since most publishers will encourage you to share the data through a data repository. The benefit of a Data Article is that your output will be {ref}`peer reviewed<cm-peer-review>`, something which is generally not the case for datasets that are archived on {ref}`data repositories<rr-rdm-sharing>`. It also facilitates recognition for datasets through research assessment procedures that are more traditionally focused on publication output. Publishing a data paper will therefore increase the visibility, credibility and usability of the data, as well as giving you credit as a data producer.
```

Author: Thank youu! Added this comment seperarately as Achintya also commented on this line.

(rr-rdm-article-options)=
## Options to publish a Data Article

> Below you can find some journals that publish data articles or papers. The costs information was collected in February 2022. 

Reviewer 2: ```suggestion
Below you can find some journals that publish data articles or papers. The costs information was collected in February 2022. 

If you are planning to publish an analysis of your published data, it may be worth checking with the journal that they wouldn't consider a data article on that dataset as 'prior publication'. F1000Research has put together a (non-exhaustive) list of journals about this [here](https://f1000research.com/data-policies?utm_source=CPB&utm_medium=cms&utm_campaign=JPF15157).
```

> ### Discipline agnostic

Reviewer: ```suggestion
### Discipline agnostic

```

| [Data in Brief](https://www.journals.elsevier.com/data-in-brief)      | [USD500](https://www.journals.elsevier.com/data-in-brief)   |
| [China Scientific Data](http://www.csdata.org/) | [RMB3000 ](http://www.csdata.org/en/p/static/1329/) ([see USD](https://www.xe.com/currencyconverter/convert/?Amount=3000&From=CNY&To=USD)) | 
| [Data Science Journal](https://datascience.codata.org/)      | [£650](https://datascience.codata.org/about/submissions/) ([see USD](https://www.xe.com/currencyconverter/convert/?Amount=650&From=GBP&To=USD)) |
| [Data](https://www.mdpi.com/journal/data)     | [CHF1.400 ](https://www.mdpi.com/journal/data/apc) ([see USD](https://www.xe.com/currencyconverter/convert/?Amount=1400&From=CHF&To=USD)) |
| [GigaScience](https://academic.oup.com/gigascience)      | [€1.089](https://academic.oup.com/gigascience/pages/charges_licensing_and_self_archiving) ([see USD](https://www.xe.com/currencyconverter/convert/?Amount=1089&From=EUR&To=USD))  |
| [Gigabyte](https://gigabytejournal.com/information-for-authors)| [USD350](https://gigabytejournal.com/open-access-and-apc#article-processing-charges) | 

> | [F1000Research](https://think.f1000research.com/about-data-notes/)      | [USD800](https://f1000research.com/for-authors/article-processing-charges)  |

Reviewer: ```suggestion
| [F1000Research](https://think.f1000research.com/about-data-notes/)      | [USD&nbsp;800](https://f1000research.com/for-authors/article-processing-charges)  |
```

| Biodiversity| [Mycokeys](https://mycokeys.pensoft.net/about#Author-Guidelines) | [variable](https://mycokeys.pensoft.net/about#Article-Processing-Charges)|
| Biodiversity| [Nature Conservation](https://natureconservation.pensoft.net/about#Author-Guidelines) | [€950](https://natureconservation.pensoft.net/about#Article-Processing-Charges)  ([see USD](https://www.xe.com/currencyconverter/convert/?Amount=950&From=EUR&To=USD)) | 
| Biodiversity| [NeoBiota](https://neobiota.pensoft.net/) | [€950](https://neobiota.pensoft.net/about#ArticleProcessingCharges)  ([see USD](https://www.xe.com/currencyconverter/convert/?Amount=950&From=EUR&To=USD)) | 
| Biodiversity| [PhytoKeys](https://phytokeys.pensoft.net/about#Author-Guidelines) |  [€780](https://phytokeys.pensoft.net/about#ArticleProcessingCharges)  ([see USD](https://www.xe.com/currencyconverter/convert/?Amount=780&From=EUR&To=USD)) |
| Biodiversity| [ZooKeys](https://zookeys.pensoft.net/about#SubmissionGuidelines) | [€780](https://zookeys.pensoft.net/about#ArticleProcessingCharges)  ([see USD](https://www.xe.com/currencyconverter/convert/?Amount=780&From=EUR&To=USD)) | 

> | Biology (plant) | [BMC Plant Biology](https://bmcplantbiol.biomedcentral.com/submission-guidelines/preparing-your-manuscript/database-article) | [€2.090](https://bmcplantbiol.biomedcentral.com/about)  ([see USD](https://www.xe.com/currencyconverter/convert/?Amount=2090&From=EUR&To=USD)) |

Reviewer: ```suggestion
| Biology (plant) | [BMC Plant Biology](https://bmcplantbiol.biomedcentral.com/submission-guidelines/preparing-your-manuscript/database-article) | [€2090](https://bmcplantbiol.biomedcentral.com/about)  ([see USD](https://www.xe.com/currencyconverter/convert/?Amount=2090&From=EUR&To=USD)) |
```

> | Biodiversity (forest)| [Annals of Forest Science](https://annforsci.biomedcentral.com/submission-guidelines/preparing-your-manuscript/data-paper) | [€1.690](https://annforsci.biomedcentral.com/submission-guidelines/fees-and-funding) ([see USD](https://www.xe.com/currencyconverter/convert/?Amount=1690&From=EUR&To=USD))| 

Reviewer: ```suggestion
| Biodiversity (forest)| [Annals of Forest Science](https://annforsci.biomedcentral.com/submission-guidelines/preparing-your-manuscript/data-paper) | [€1690](https://annforsci.biomedcentral.com/submission-guidelines/fees-and-funding) ([see USD](https://www.xe.com/currencyconverter/convert/?Amount=1690&From=EUR&To=USD))| 
```

> | Botany | [Botanical Studies](https://as-botanicalstudies.springeropen.com/submission-guidelines/preparing-your-manuscript/database-article) | [€1.745](https://as-botanicalstudies.springeropen.com/about)  ([see USD](https://www.xe.com/currencyconverter/convert/?Amount=1745&From=EUR&To=USD)) | 

Reviewer: ```suggestion
| Botany | [Botanical Studies](https://as-botanicalstudies.springeropen.com/submission-guidelines/preparing-your-manuscript/database-article) | [€1745](https://as-botanicalstudies.springeropen.com/about)  ([see USD](https://www.xe.com/currencyconverter/convert/?Amount=1745&From=EUR&To=USD)) | 
```

| Chemistry | [Journal of Cheminformatics](https://jcheminf.biomedcentral.com/submission-guidelines/preparing-your-manuscript/data-note) | [0](https://jcheminf.biomedcentral.com/about) |
| Computer Science| [Jaiio](https://www.sadio.org.ar/jaiio/)| [variable](https://50jaiio.sadio.org.ar/aranceles) | 

> | Ecology | [Ecological Research](http://www.esj.ne.jp/er/datapaper.html) | [USD3.000](https://esj-journals.onlinelibrary.wiley.com/hub/journal/14401703/homepage/forauthors) |

Reviewer: ```suggestion
| Ecology | [Ecological Research](http://www.esj.ne.jp/er/datapaper.html) | [USD&nbsp;3000](https://esj-journals.onlinelibrary.wiley.com/hub/journal/14401703/homepage/forauthors) |
```

| Ecology | [Global Ecology and Biogeography](https://onlinelibrary.wiley.com/page/journal/14668238/homepage/forauthors.html) | [€3.050](https://authorservices.wiley.com/author-resources/Journal-Authors/open-access/article-publication-charges.html)  ([see USD](https://www.xe.com/currencyconverter/convert/?Amount=3050&From=EUR&To=USD))| 
| Earth Sciences| [Geoscience Data Journal](https://rmets-onlinelibrary-wiley-com.tudelft.idm.oclc.org/journal/20496060)      | [€1.450](https://rmets.onlinelibrary.wiley.com/hub/journal/20496060/article-publication-charge)  ([see USD](https://www.xe.com/currencyconverter/convert/?Amount=1450&From=EUR&To=USD))  |
| Earth Sciences| [Earth System Science Data](https://www.earth-system-science-data.net/)    | 0  |
| Earth Sciences| [Big Earth Data](https://www.tandfonline.com/action/authorSubmission?show=instructions&journalCode=tbed20) | [€910](https://www.tandfonline.com/action/authorSubmission?show=instructions&journalCode=tbed20&#apc)  ([see USD](https://www.xe.com/currencyconverter/convert/?Amount=910&From=EUR&To=USD)) |

> | Ecology | [BMC Ecology and Evolution](https://bmcecolevol.biomedcentral.com/submission-guidelines/preparing-your-manuscript/database-article) | [€2.090](https://preview-bmcecolevol.biomedcentral.com/submission-guidelines/fees-and-funding)  ([see USD](https://www.xe.com/currencyconverter/convert/?Amount=2090&From=EUR&To=USD)) | 

Reviewer: ```suggestion
| Ecology | [BMC Ecology and Evolution](https://bmcecolevol.biomedcentral.com/submission-guidelines/preparing-your-manuscript/database-article) | [€2090](https://preview-bmcecolevol.biomedcentral.com/submission-guidelines/fees-and-funding)  ([see USD](https://www.xe.com/currencyconverter/convert/?Amount=2090&From=EUR&To=USD)) | 
```
